src/routes/comment.js

```diff
@@ -0,0 +1,68 @@
+
+import {Router} from 'express';
+import {CreateComment, GetComment} from '../controllers/comment';
+import auth from '../middlewares/check-auth';
+
+const router  = Router();
+
+/**
+ * @swagger
+ * /{requestId}/comments:
+ *    get:
+ *      summary: read Comments
+ *      tags: [Comments]
+ *      parameters:
+ * 
+ *        - name: requestId
+ *          in: path
+ *          type: string
+ *          required: true
+ * 
+ *      responses:
+ *        "200":
+ *          description: Receive Comments
+ *        "500":
+ *          description: Error  
+ */
+
+router.get('/:requestId/comments', auth, GetComment);
```

Comment:
About this **url** format, can anyone see a comment of an imagined request? or you have to see the request and related comments.

Reply:
yes ResquestId for now is  imagined because I don't have the real request

Reply:
Yes that why need **travel request for jemima'work**